### PR TITLE
DbSnpBitSetUtil should verify that the VCF dict should match the inpu…

### PR DESCRIPTION
**Pull request by nh13**
…t dict

Feel free to push directly to this, for example fixes and tests.  I make this PR for demonstration purposes for #1631

### CODE REVIEW

**Comment by kockan** [comment]
Hi @nh13 , I made some small changes for this patch to successfully pass the checks and added some basic tests for the pass/fail cases (hg19 vs hg19 and hg19 vs hg38). Does it seem reasonable to you?

File: src/main/java/picard/util/DbSnpBitSetUtil.java [hunk 1]
```diff
@@ -213,6 +214,14 @@ private static void loadVcf(final PicardHtsPath dbSnpFile,
 
         final Optional<ProgressLogger> progress = log.map(l -> new ProgressLogger(l, (int) 1e5, "Read", "variants"));
         final VCFFileReader variantReader = new VCFFileReader(dbSnpFile.toPath(), intervals != null);
+        
+        SequenceDictionaryUtils.assertSequenceDictionariesEqual(
+            variantReader.getFileHeader().getSequenceDictionary(),
+            "DBSNP_REFERENCE_VCF",
+            sequenceDictionary,
+            "INPUT_VCF"
```
Comment (nh13) [inline src/main/java/picard/util/DbSnpBitSetUtil.java:222]: I think this is fine to list the command line parameter names in an internal function

**Comment by nh13** [comment]
lgtm

File: src/main/java/picard/util/DbSnpBitSetUtil.java (hunk 1 above)
Comment (kockan) [reply src/main/java/picard/util/DbSnpBitSetUtil.java]: Done.

**Review by kockan — APPROVED** [review]
Ready to go.

**Pull request merged**